cmd/nf-dump/main.go

```diff
@@ -40,16 +40,21 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if err = server.SetReadBuffer(262144); err != nil {
```

Comment:
This call is very important for high performance udp servers in Go. It seems Go doesn't pull datagrams off the buffer quickly enough (in terms of latency) before the buffer fills and some datagrams are dropped.

What OS were you using for this value? On Linux, the value of `/proc/sys/net/core/rmem_max` will control how large you can set the read buffer, and the default is generally 100-200k. My Centos 7 box reports 212992 bytes. To muddy the waters more, the kernel actually *doubles* the value you provide when setting the socket option `SO_RCVBUF`, which is what Go does via `SetReadBuffer()`. See `SO_RCVBUF` in http://man7.org/linux/man-pages/man7/socket.7.html